requirements-dev.txt

```diff
@@ -10,9 +10,9 @@ pyzmq
 setuptools
 twine
 # Dependencies for linting. Versions match those in setup.py.
-flake8==5.0.2
+flake8>=5.0.2
 clang-format==13.0.0
-mypy==0.782
+mypy>=0.782
```

Comment:
Shall we bump the used mypy version in [setup.py](https://github.com/onnx/onnx/blob/5fbb7b507e8c908d210179181c9f22ffc01e80fc/setup.py#L333) since this PR enable mypy check with mypy 0.971?

And I think this PR can close this issue: https://github.com/onnx/onnx/issues/3964.

Reply:
done